tools/zod2md-jsdocs/project.json

```diff
@@ -4,20 +4,8 @@
   "sourceRoot": "tools/zod2md-jsdocs/src",
   "projectType": "library",
   "targets": {
-    "build": {
-      "executor": "@nx/js:tsc",
-      "outputs": ["{options.outputPath}"],
-      "dependsOn": ["pre-build"],
-      "options": {
-        "outputPath": "tools/zod2md-jsdocs/dist",
-        "main": "tools/zod2md-jsdocs/src/index.ts",
-        "tsConfig": "tools/zod2md-jsdocs/tsconfig.lib.json"
-      }
-    },
-    "pre-build": {
-      "command": "ts-patch install",
-      "cache": true,
-      "inputs": ["sharedGlobals", { "runtime": "ts-patch check" }]
-    }
```

Comment:
I would restore this target - see [previous comment](https://github.com/code-pushup/cli/pull/1162/changes#r2610811720). I'm fine with renaming `pre-build` to `tspatch`, though.